nyaa/views/torrents.py

```diff
@@ -40,7 +40,16 @@ def view_torrent(torrent_id):
 
         if comment_form.validate():
             comment_text = (comment_form.comment.data or '').strip()
-
+            mentioned = [s for s in comment_text.split() if '@' in s]
+            if len(mentioned) > 0:
+                username = mentioned[0][1:]
```

Comment:
It's worth mentioning is that you only use the first username in the comment.
I think you should use all of them.
Obviously we don't want any duplicates, so use something like:
```python
mentioned = [s[1:] for s in comment_text.split() if '@' in s]
# ...skipping...
for username in set(mentioned):  # sets only allow unique elements
# continue code...
```